Stripe/STPMandateOnlineParams.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  STPMandateOnlineParams.swift
+//  Stripe
+//
+//  Created by Cameron Sabol on 10/17/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+import Foundation
+
+/// Contains details about a Mandate accepted online. - seealso: https://stripe.com/docs/api/payment_intents/confirm#confirm_payment_intent-mandate_data-customer_acceptance-online
+public class STPMandateOnlineParams: NSObject {
+
+  /// The IP address from which the Mandate was accepted by the customer.
+  @objc public let ipAddress: String
+
+  /// The user agent of the browser from which the Mandate was accepted by the customer.
+  @objc public let userAgent: String
+
+  @objc public var additionalAPIParameters: [AnyHashable: Any] = [:]
+
+  @objc internal var inferFromClient: NSNumber?
+
+  @objc(initWithIPAddress:userAgent:)
+  init(ipAddress: String, userAgent: String) {
+    self.ipAddress = ipAddress
+    self.userAgent = userAgent
+    super.init()
+  }
+}
+
+extension STPMandateOnlineParams: STPFormEncodable {
+
+  @objc internal var ipAddressField: String? {
+    guard inferFromClient == nil || !(inferFromClient?.boolValue ?? false) else {
+      return nil
+    }
+    return ipAddress
+  }
+
+  @objc internal var userAgentField: String? {
```

Comment:
@csabol-stripe This `inferFromClient` check seems to be new behavior: That's intentional, right?